### PR TITLE
FIX: Avoid copying node modules

### PR DESCRIPTION
**Pull request by eduhdm**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,9 +3,9 @@ FROM node:12-slim
 WORKDIR /usr/src/app
 ENV NODE_ENV=production
 
-COPY package*.json ./
-RUN npm install --only=production
 COPY . ./
+RUN npm install --only=production
```
Comment (marco-souza) [inline Dockerfile:7]: We should also ignore `node_modules`  volume on docker-compose file, in case we have one.

File: Dockerfile [hunk 2]
```diff
@@ -3,9 +3,9 @@ FROM node:12-slim
...
+
 RUN npm run build
 
 CMD [ "npm", "start" ]
```
Comment (marco-souza) [inline Dockerfile]: add end line

File: .dockerignore [hunk 3]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (marco-souza) [inline .dockerignore]: add end line

Comment (matheusaraujo) [reply .dockerignore]: wright!

File: Dockerfile (hunk 1 above)
Comment (matheusaraujo) [reply Dockerfile:7]: good!

File: Dockerfile (hunk 2 above)
Comment (matheusaraujo) [reply Dockerfile]: wright

File: Dockerfile (hunk 1 above)
Comment (eduhdm) [reply Dockerfile:7]: it's implemented in https://github.com/cantodaruaemergencial/infra/tree/feat/setup-compose-for-application docker-compose.

thanks!

**Review by marco-souza — APPROVED** [review]
I'll let this approved, but I ain't sure if the `.dockerignore`file is considered by docker-compose, when its mounting volumes.

**Pull request merged**